openfl-tutorials/interactive_api/PyTorch_Market_Re-ID/workspace/tools.py

```diff
@@ -102,7 +109,8 @@ def evaluate(distmat, q_pids, g_pids, q_camids, g_camids):
 
 
 @torch.no_grad()
-def extract_feature(model, dataloader):
+def extract_feature(model: Any,
+                    dataloader: Any) -> Tuple[torch.tensor, torch.tensor, torch.tensor]:
```

Comment:
`dataloader` can be iterable yielding tuples, but we pass here only PyTorch DataLoaders.

Reply:
Do not use functions as types.